content/01.abstract.md

```diff
@@ -4,10 +4,7 @@ Important tasks in biomedical discovery such as predicting gene functions, gene-
 The number of edges connecting to a node, termed degree, can vary greatly across nodes in real biomedical networks, and the distribution of degrees varies between networks.
 If degree strongly influences edge prediction, then imbalance or bias in the distribution of degrees could lead to nonspecific or misleading predictions.
 We introduce a network permutation framework to quantify the effects of node degree on edge prediction.
-Our framework decomposes performance into the proportions attributable to degree and the network's specific connections.
+Our framework decomposes performance into the proportions attributable to degree and the network's specific connections using network permutation to generate features that depend only on degree.
 We discover that performance attributable to factors other than degree is often only a small portion of overall performance.
-Degree's predictive performance diminishes when the networks used for training and testing---despite measuring the same biological relationships---were generated using distinct techniques and hence have large differences in degree distribution.
-We introduce the permutation-derived edge prior as the probability that an edge exists based only on degree.
-The edge prior shows excellent discrimination and calibration for 20 biomedical networks (16 bipartite, 3 undirected, 1 directed), with AUROCs frequently exceeding 0.85.
-Researchers seeking to predict new or missing edges in biological networks should use the edge prior as a baseline to identify the fraction of performance that is nonspecific because of degree.
+Researchers seeking to predict new or missing edges in biological networks should use our permutation approach to obtain a baseline for performance that may be nonspecific because of degree.
```

Comment:
```suggestion
Researchers seeking to predict new or missing edges in biological networks can use our permutation approach to obtain a baseline for performance that may be nonspecific because of degree.
```